package/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "labeling-tool",
-  "version": "2.0.5",
+  "version": "2.0.7",
```

Comment:
deploy to the NMP registry, update in the core FE and test there, if it works as expected - will merge